test-it/karma/utilities/MainAmd.js

```diff
@@ -4,7 +4,7 @@
     var tests = [];
     for (var file in window.__karma__.files) {
         if (window.__karma__.files.hasOwnProperty(file)) {
-            if (/test\.js$/.test(file)) {
+            if ((/test\.js$/).test(file)) {
```

Comment:
What is the reason behind this change? Is this protecting against a vulnerability?

Reply:
Nah it was an eslint rule that changed in eslint 5 to make regular expressions unambiguous by wrapping them in parens